src/views/StackView/createPointerEventsContainer.tsx

```diff
@@ -84,7 +84,17 @@ export default function createPointerEventsContainer<
         return 'box-only';
       }
 
-      return 'auto';
+      const gesturesEnabled = this.props.scene.descriptor.options
```

Comment:
This should be passed down instead of calculating here

Reply:
I initially wanted to pass "box-none" here instead of "auto" without even taking "gesturesEnabled" into consideration. Wouldn't that be a better option?
I would like to avoid adding additional logic for this into <Component /> (i.e. StackViewCard) as it would break the single-responsibility (i.e. createPointerEventsContainer is to take care of setting up the pointerEvents so why wouldn't it do it fully and move part of the logic into StackViewCard?).

What do you think?